### PR TITLE
Frontend structure

### PR DESCRIPTION
**Pull request by alfonsomartinezs**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,43 @@
...
+import mockRocks from "./mockRocks";
+
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rocks: mockRocks,
+    };
```
Comment (sjproctor) [inline src/App.js:23]: Mock rocks makes me laugh.

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import "../App.css";
```
Comment (sjproctor) [inline src/components/Footer.js]: I would move this to App.js so you don't have to import it on all the files.

**Review by sjproctor — APPROVED** [review]
Nice work!

File: src/components/Footer.js (hunk 2 above)
Comment (alfonsomartinezs) [reply src/components/Footer.js]: thank you! made this change

File: src/App.js (hunk 1 above)
Comment (alfonsomartinezs) [reply src/App.js:23]: Us too!

**Pull request merged**